actors/v2/account/account.go

```diff
@@ -98,3 +113,9 @@ func (a *Account) UniversalReceiverHook(network string, height int64, raw []byte
 	metadata[parser.ParamsKey] = base64.StdEncoding.EncodeToString(raw)
 	return metadata, nil
 }
+
+func (a *Account) Fallback(network string, height int64, raw []byte) (map[string]interface{}, error) {
+	metadata := make(map[string]interface{})
+	metadata[parser.ParamsRawKey] = raw
```

Comment:
Shouldn't we hex string here too? I saw we do it on other cases, but here we don't... we should do it same way to homogeneousness it